components/definition/cellRules/NetPyNECellRules.js

```diff
@@ -47,7 +47,7 @@ export default class NetPyNECellRules extends React.Component {
   }
 
   handleNewCellRule() {
-    var defaultCellRules = { 'CellRule': {'conds':{}, 'secs':{}} }
+    var defaultCellRules = { 'CellRule': {'conds':{'cellType': "", 'cellModel': ""}, 'secs':{}} }
```

Comment:
@rodriguez-facundo is this needed for fixing the tests? Just noticing you are checking for null.

Reply:
You are right... *cellModel* and *cellType* are null now until the user selects a value. So maybe I could just remove from the test, the checking for these params.